enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+
+#### OCM Standalone Standard Change Management Scenario
+
+1. User interactions with OCM to configure a maintenance schedule are identical to [OCM HCP Standard Change Management Scenario](#ocm-hcp-standard-change-management-scenario).
+   This scenario differs after OCM accepts the maintenance schedule configuration. Control-plane updates are permitted to be initiated to any Saturday UTC.
+   Worker-nodes must wait until the first Saturday of the month.
+1. OCM (through various layers) configures the ClusterVersion and worker MachineConfigPool(s) (MCP) for the cluster with appropriate `changeManagement` stanzas.
+1. Company workloads are added to the new cluster and the cluster provides value.
+1. To leverage a new feature in OpenShift, the service consumer plans to update the minor version of the platform.
+1. Via OCM, the service consumer requests the minor version update. They can do this at any time with confidence that the maintenance 
+   schedule will be honored. They do so on Wednesday.
+1. OCM (through various layers) updates the ClusterVersion resource on the cluster indicating the new release payload in `desiredUpdate`.
+1. The Cluster Version Operator (CVO) detects that its `changeManagement` stanza does not permit the initiation of the change.
+1. The CVO sets a metric indicating that changes are pending for ClusterVersion. Irrespective of pending changes, the CVO also exposes a 
+   metric indicating the number of seconds until the next window in which material changes can be initiated.
+1. Since MachineConfigs do not match in the desired update and the current manifests, the CVO also sets a metric indicating that MachineConfig
+   changes are pending. This is done because the MachineConfigOperator (MCO) cannot anticipate the coming manifest changes and cannot,
+   therefore, reflect expected changes to the worker-node MCPs. Anticipating this change ahead of time is necessary for an operation
+   team to be able to set an alert with the semantics (worker-node-update changes are pending & time remaining until changes are permitted < 2d).
+   The MCO will expose its own metric for changes pending when manifests are updated. But this metric will only indicate when 
+   there are machines in the pool that have not achieved the desired configuration. An operations team trying to implement the 2d 
+   early warning for worker-nodes must use OR on these metrics to determine whether changes are actually pending.
+1. The MCO, irrespective of pending changes, exposes a metric for each MCP to indicate the number of seconds remaining until it is
+   permitted to initiate changes to nodes in that MCP.
+1. A privileged user on the cluster notices different options available for `changeManagement` in the ClusterVersion and MachineConfigPool
+   resources. They try to set them but are prevented by either RBAC or an admission webhook (details for Service Delivery). If they wish
```

Comment:
```suggestion
   resources. They try to set them but are prevented by validating admission controller. If they wish
```

Customers can use `cluster-admin` so blocking critical resource edits is done via admission controller.